std/json.d

```diff
@@ -628,7 +655,7 @@ struct JSONValue
     /// Implicitly calls $(D toJSON) on this JSONValue.
     ///
     /// $(I options) can be used to tweak the conversion behavior.
-    string toString(in JSONOptions options = JSONOptions.none) const
+    string toString(in JSONOptions options = JSONOptions.none) const @safe
     {
         return toJSON(&this, false, options);
```

Comment:
Changed to ref and added a deprecated backwards compatibility pointer overload. @safe functions aren't allowed to take the address of a ref parameter, so there's no risk of pointers escaping.

There's still no deprecation policy that I know of, so I'm putting it at 18 months provisionally.